packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -139,6 +143,9 @@ export default class TradeStore extends BaseStore {
     // Mobile
     is_trade_params_expanded = true;
 
+    //Toastbox
+    action_toastbox = {};
```

Comment:
`action_toastbox` doesn't convey the purpose of this prop. Can we consider `contract_purchase_toast_box`?